packages/blockchain-wallet-v4-frontend/src/data/components/exchange/types.ts

```diff
@@ -34,20 +34,25 @@ export type LimitDurationType = {
   symbol: string
 }
 
-export type SourceFeeType = {
-  isSourceErc20: boolean
-  mempoolFees: {
-    limits: {
-      max: number
-      min: number
+export type SourceFeeType =
+  | {
+      isSourceErc20: boolean
+      mempoolFees: {
+        limits: {
+          max: number
+          min: number
+        }
+        priority: number
+        regular: number
+      }
+      source: number | string
+      sourceFiat: string
+      target: number | string
+    }
+  | {
```

Comment:
I added this to source fee type. Everything worked, was able to make an exchange.